extensions/wikia/AdEngine/js/video/videoAdFactory.js

```diff
@@ -0,0 +1,60 @@
+/* global define */
+define('ext.wikia.adEngine.video.videoAdFactory', [
+	'ext.wikia.adEngine.domElementTweaker',
+	'ext.wikia.adEngine.video.googleIma',
+	'ext.wikia.adEngine.video.vastUrlBuilder',
+	'wikia.log'
+], function (DOMElementTweaker, googleIma, vastUrlBuilder, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.video.videoAdFactory';
+
+	function init() {
+		return googleIma.init();
+	}
+
+	function create(imageContainer, getSlotWidth, getSlotHeight, adSlot, slotParams, onVideoEndedCallback) {
```

Comment:
Consider passing a parameterized object into the `create()` method instead of so many parameters.